packages/selfcare-client-users-updater-consumer/package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "pagopa-interop-selfcare-client-users-updater-consumer",
+  "private": true,
+  "version": "1.0.0",
+  "description": "PagoPA Interoperability Selfcare client users updater consumer: a Kafka consumer that listens to messages from the selfcare client users updater topic and executes the corresponding actions.",
```

```suggestion
  "description": "PagoPA Interoperability Selfcare Client Users Updater: a Kafka consumer that listens to messages from the Selfcare 'SC-Users' topic and updates client users based on the received messages",
```
I would keep it more generic since in the future it will handle other users too